app/src/main/java/com/launcher/silverfish/launcher/homescreen/HomeScreenFragment.java

```diff
@@ -595,6 +651,55 @@ public boolean onTouch(View view, MotionEvent event) {
 
     //endregion
 
+    //region Extra options per app
+
+    /** Home screen 'pop-up' menu after minimal drag */
+    private void showExtraOptionsMenu(final String appId, final String appIndex, final String appName, String appLabel) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+
+        // Set up a title e.g. "Silverfish (com.launcher.silverfish)"
+        String title = format("<font color='red'><b>%s</b></font> ", appLabel);
```

Comment:
Same here, not only hardcoding the color used by also putting its definition in two different places one must remember.